asllib/menhir2bnfc/tests/grammars.t/calc.bnf.expected

```diff
@@ -1,4 +1,8 @@
-// entrypoints Main;
+-- DISCLAIMER: This file has been autogenerated by menhir2bnfc.
+-- This file is **NOT** a reference implementation and should not be treated as such!
+-- Modifications of this file are an **UNSUPPORTED** divergence from the reference implementation used to generate it and should **NOT** be done!
+
```

Comment:
please use the same wording as elsewhere, viz:

Modifications of this file will lead to an **UNSUPPORTED** divergence from the reference implementation and should **NOT** be done!